assignment.js

```diff
@@ -176,7 +185,15 @@ assert.equal(nextExitLabel(),  "exit-1")
 
 function labelMaker1(label) {
   // START
-  // ...
+    let count = 0
+    return function (num) {
+        if (num != undefined) {
+            count = num
+        }
+
+        return label + "-" + count++
```

Comment:
I'm surprised that you used `count++` here but did the somewhat ugly `count=-1...count++; return` above...